assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -101,143 +109,99 @@ class TextBlockEdit extends Component {
 				} :
 				block
 		) ) );
-	}
-
-	toggleIsEditing = ( enable ) => {
-		if ( enable !== this.state.isEditing ) {
-			this.setState( {
-				isEditing: ! this.state.isEditing,
-			} );
-		}
-	}
-
-	toggleOverlay = ( add ) => {
-		if ( add !== this.state.hasOverlay ) {
-			this.setState( {
-				hasOverlay: ! this.state.hasOverlay,
-			} );
-		}
-	}
-
-	render() {
-		const { isEditing, hasOverlay } = this.state;
+	};
 
-		const {
-			attributes,
-			setAttributes,
-			className,
-			clientId,
-			fontSize,
-			isPartOfMultiSelection,
-			isSelected,
-			backgroundColor,
-			customBackgroundColor,
-			textColor,
-			name,
-		} = this.props;
+	const userFontSize = fontSize && fontSize.size ? fontSize.size + 'px' : undefined;
 
-		const {
-			placeholder,
-			content,
-			align,
-			ampFitText,
-			autoFontSize,
-			height,
-			opacity,
-		} = attributes;
+	const { colors } = select( 'core/block-editor' ).getSettings();
```

Comment:
No `useSelect` here? One reason would be to force a re-render if the store changes.

Reply:
I just missed that one I guess :-)

Reply:
Fixed now